README.md

```diff
@@ -119,39 +132,33 @@ The database seed contains most of the custom configuration, theme selection, ad
 
 ### 7. Set Magento to `production` mode
 
-The explicit call to `setup:static-content:deploy` is required due to requiring multiple locales.  `en_AU` is used in
-the frontend, and `en_US` is used for the backend admin UI.  The final command sets Magento to production mode, 
-including compiling the dependency injections and other configuration.
+In any deployed environment, Magento should be in `production` mode, as opposed to `default` or `developer`.  
 
 ```
-bin/magento setup:static-content:deploy en_AU en_US
```

Comment:
One problem with this approach is with session timeouts.  The user is redirected back to the login screen without the locale setting, which is broken.

However I think this is better than maintaining two whole copies of the entire static content just for one page.  Could be convinced otherwise.